### DISCUSSION TITLE
SideMenu: Pin folders or dashboards to side menu (with custom icon/image)

### DISCUSSION THREAD

**Discussion by tomwilkie** [grafana/grafana#38372] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Can I set per-folder icons to be displayed in the nav bar please?

**Why is this needed**:

Honestly, I think it will look nice.  It might also help people find things quicker; but mostly I think it'll look nice.

**Comment by torkelo** [comment]
I think this is a really good idea, pining either dashboards or folders to the side menu with a custom icon/image. 

It would be user preference or would you want it on org level as well?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
